### PR TITLE
Modulesync

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
+    concat_basedir: '/tmp',
+    is_pe: false,
+    selinux_config_mode: 'disabled',
+    puppetversion: Puppet.version,
+    facterversion: Facter.version,
+    ipaddress: '172.16.254.254',
+    macaddress: 'AA:AA:AA:AA:AA:AA'
+  }
 end
+require 'rspec/mocks/standalone'
```
Comment (bastelfreak) [inline spec/spec_helper.rb]: is this still needed?

File: Gemfile [hunk 2]
```diff
@@ -11,38 +11,42 @@ def location_for(place, fake_version = nil)
...
-  gem 'beaker-rspec', '5.4.0'
+  gem 'beaker', '2.43.0',              :require => false
+  if beaker_version = ENV['BEAKER_VERSION']
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```
Comment (bastelfreak) [inline Gemfile]: this whole beaker block looks wrong :(

File: spec/spec_helper.rb (hunk 1 above)
Comment (petems) [inline spec/spec_helper.rb]: Tests seem to pass with or without it...

File: Gemfile (hunk 2 above)
Comment (petems) [inline Gemfile]: Hmm, how does the gemfile get the versions correctly then? 😢

File: spec/spec_helper.rb (hunk 1 above)
Comment (bastelfreak) [inline spec/spec_helper.rb]: ah yeah. I removed it now

File: Gemfile (hunk 2 above)
Comment (bastelfreak) [inline Gemfile]: @petems as a workaround: we could set the version in modulesync_config?

Comment (bastelfreak) [inline Gemfile]: I just noticed that our default Gemfile is a bit wrong:
```ruby
group :system_tests do
  gem 'beaker',                        :require => false
  if beaker_version = ENV['BEAKER_VERSION']
    gem 'beaker', *location_for(beaker_version)
  end 
  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
    gem 'beaker-rspec', *location_for(beaker_rspec_version)
  else
    gem 'beaker-rspec',  :require => false
  end 
  gem 'beaker-puppet_install_helper',  :require => false
end
```

we already include beaker twice if an ENV var is set. So we can stay with the changes in this PR?

**Comment by petems** [comment]
Discussed with @bastelfreak, the modulesync gem issue is an upstream config issue. Tests and specs still pass and I dont want to block this on an upstream issue, so LGTM 👍

**Pull request merged**